docusaurus/docs/iOS/components/message-list.md

```diff
@@ -1,3 +1,272 @@
 ---
 title: Message List
----
\ No newline at end of file
+---
+
+import ComponentsNote from '../common-content/components-note.md'
+import MessageListProperties from '../common-content/reference-docs/stream-chat-ui/chat-message-list/chat-message-list-vc-properties.md'
+
+The `ChatMessageListVC` is a component responsible to display a list of messages. It is not the responsibility of the message list to know the source of the messages, the data should be provided through the `ChatMessageListVCDataSource` and the `ChatMessageListVCDelegate`, very similar to how a native `UITableView` and a `UICollectionView` works.
```

Comment:
I suggest making this a positive sentence, say what it is and not what is it not :)